.ci/deps.sh

```diff
@@ -105,3 +105,9 @@ fi
 # Tailor (Swift) commands
 curl -fsSL https://tailor.sh/install.sh | sed 's/read -r CONTINUE < \/dev\/tty/CONTINUE=y/' > install.sh
 sudo bash install.sh
+
+# making coala cache the dependencies downloaded upon first run
+echo '' > dummy
+coala-ci --bears CheckstyleBear --files dummy --no-config || true
```

Comment:
@AbdealiJK I tried /dev/null first, but coala couldn't check it since it's
outside the working directory.

`|| true` is there to make sure the build doesn't get interrupted if a bear
returns an error for an empty file. (For instance YamlLintBear I think
would complain that the front matter is missing.) The purpose of the
command is to get the deps and that would be done regardless of exit code.

On Tue, Sep 6, 2016, 9:45 AM AbdealiJK <notifications@github.com> wrote:

> In .ci/deps.sh
> <https://github.com/coala-analyzer/coala-bears/pull/767#discussion_r77586441>
> :
>
> > @@ -105,3 +105,9 @@ fi
> >  # Tailor (Swift) commands
> >  curl -fsSL https://tailor.sh/install.sh | sed 's/read -r CONTINUE < \/dev\/tty/CONTINUE=y/' > install.sh
> >  sudo bash install.sh
> > +
> > +# making coala cache the dependencies downloaded upon first run
> > +echo '' > dummy
> > +coala-ci --bears CheckstyleBear --files dummy --no-config || true
>
> Use /dev/null instead (Minor)
>
> Also, why || true ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/767/files/55e729232ff4e4a5faecfc975d4dfe82d4610033..e9c0a5c3b42a73c39be0c3d6081c3a59101867c4#r77586441>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABAuVPtv1ur7jM17NzNe9KUko9Ut3sBEks5qnRoSgaJpZM4JzMuc>
> .
>